### PR TITLE
renaming and consolidating tasks

### PR DESCRIPTION
**Pull request by gvn**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -13,6 +13,20 @@ npm install
...
 ```
 
+## Tasks
+
+- `npm build` Build project.
+- `npm run js-beautify` Run JS Beautifier on all JS files.
+- `npm run jscs` Run JSCS on all JS files.
+- `npm run jshint` Run JSHint on all JS files.
+- `npm run validate` Ensure all JS files pass JSHint and JSCS rules defined in [mofo-style](https://github.com/MozillaFoundation/javascript-style-guide).
+- `npm run watch` Build LESS and JS files when they change.
```
Comment (alicoding) [inline README.md]: lets remove this? since we are going with `start:dev`

File: package.json [hunk 2]
```diff
@@ -4,16 +4,17 @@
...
-    "jscs": "jscs lib test web *.js",
     "js-beautify": "js-beautify  lib test web *.js -r ",
-    "validate": "npm run jshint && npm run jscs"
+    "jscs": "jscs lib test web *.js",
+    "jshint": "jshint  lib test web *.js",
+    "start": "node web/index.js",
+    "start:dev": "parallelshell \"npm start\" \"npm run watch\"",
+    "test": "lab -t 100 && npm run validate",
+    "validate": "npm run jshint && npm run jscs",
+    "watch": "parallelshell \"npm run watch:css\" \"npm run watch:js\"",
```
Comment (alicoding) [inline package.json]: also this

**Comment by alicoding** [comment]
Just remove that one and R+ :+1: as long as Travis is happy then lets land this! :dancer:

**Comment by gvn** [comment]
@k88hudson Can you let me know when you've landed your new tasks? I'll hold off merging until then...

**Comment by k88hudson** [comment]
@gvn yep, almost done. thanks! i really appreciate it :+1:

**Comment by jbuck** [comment]
This PR needs to be redone against develop

**Comment by jbuck** [comment]
Closed by #89

**Pull request closed without merging**